src/netbase.cpp

```diff
@@ -826,14 +826,11 @@ bool LookupSubNet(const std::string& strSubnet, CSubNet& ret)
         return false;
     }
     size_t slash = strSubnet.find_last_of('/');
-    std::vector<CNetAddr> vIP;
+    CNetAddr network, netmask;
```

Comment:
The scope of `netmask` could be reduce to make review somewhat easier?

Reply:
Good idea! Done.